weblate_schemas/message.py

```diff
@@ -0,0 +1,106 @@
+# Copyright © Michal Čihař <michal@weblate.org>
+#
+# SPDX-License-Identifier: MIT
+
+"""Message class implementation for Weblate Fedora Messaging."""
+
+from datetime import datetime
+
+from weblate_schemas import load_schema
+from fedora_messaging import message
+
+
+class BaseMessage(message.Message):
+    """Inherit the Message class from fedora_messaging."""
+
+    def __init__(self, **kwargs) -> None:
+        """Initialize BaseMessage class."""
+        super().__init__(**kwargs)
+
+    @property
+    def app_name(self) -> str:
+        """Return the app name."""
+        return "Weblate"
+
+    @property
+    def app_icon(self) -> str:
+        """Return the App icon URL."""
+        return "https://weblate.org/static/weblate-128.png"
+
+
+class WeblateMessage(BaseMessage):
```

Comment:
Probably should include versioning to allow future changes:

```suggestion
class WeblateV1Message(BaseMessage):
```

Reply:
You don't need to have it for the initial message schema, but will need that for any updated message schema. The first one could be `WeblateMessage`, but the changed one would need to be `WeblateV2Message`. Still a good idea to have the versioning in from start.

Keep in mind that you only need to do that if you are removing something from the message schema, adding things doesn't break anything.